docs/cmdline/create.md

```diff
@@ -0,0 +1,60 @@
+---
+title: terramate create - Command
+description: With the terramate create command you can create a new stack in the current project.
+
+prev:
+  text: 'Stacks'
+  link: '/stacks/'
+
+next:
+  text: 'Sharing Data'
+  link: '/data-sharing/'
+---
+
+# Create
+
+The `create` command creates a new stack. This command accepts a path that will
+be used to create the stack in. The `create` command will create intermediate
+directories as required.
+
+## Usage
+
+Usage:
+
+`terramate create [path] [options]`
```

Comment:
@mariux is this correct or shall we rather say `terramate create [options] [path]` (which is what Terraform is using)

Reply:
i would even say:

```suggestion
`terramate create [options] PATH`
```

as PATH is not optional bu required.